code/cogs/chat.py

```diff
@@ -37,16 +39,23 @@ async def on_message_process(message: discord.Message, self):
         return
     #check if the message has been sent in the channel set in the database
     c.execute("SELECT channel_id FROM data WHERE guild_id = ?", (message.guild.id,))
+    #select channels from the database
+    cp.execute("SELECT * FROM channels WHERE guild_id = ?", (message.guild.id,))
```

Comment:
Shouldn't you here do a `try except`? Because it looks like in the case that the case when the guild isn't premium, it's not gonna work.

Reply:
the code will just return an error

Reply:
that's fixed in the commit i just published